configure.ac

```diff
@@ -717,13 +721,16 @@ AC_SUBST(OLA_CLIENT_LIBS)
 
 # Extra tools
 #####################################################
-AM_CONDITIONAL([BUILD_JA_RULE],
-               [test "x$enable_ja_rule" = xyes && test "x$have_libusb" = xyes \
-                && test "x$have_libusb_hotplug_api" = xyes])
+AS_IF([test "x$enable_ja_rule" = xyes && test "x$enable_usbdmx" != xyes],
+      AC_MSG_ERROR([Ja Rule requires the usbdmx plugin to be enable (--enable-usbdmx).]))
+AS_IF([test "x$enable_ja_rule" = xyes && test "x$have_libusb_hotplug_api" != xyes],
```

Comment:
I'd like to be belt and braces and keep this check in too "x$have_libusb" = xyes.